.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: goreleaser
+
+on:
+  push:
```

Comment:
When do you want it to be triggered ? Currently it is going to be run on every `push` for every branch.

We can trigger it when:
1. Maintainer calls it via [actions](https://github.com/scylladb/gemini/actions) tab
```
on:
  workflow_dispatch
```
2. Make it triggered automagically if PR have label is being merged:
```
on:
  pull_request:
    branches: 
      - main
    types: [closed]

jobs:
  my-action:
    if: ${{ github.event.pull_request.merged }}
```

Reply:
that's not correct, read the next line, it's limited to tag pushes only